components/builder-api/src/server/resources/origins.rs

```diff
@@ -278,10 +277,6 @@ fn delete_origin(req: HttpRequest<AppState>) -> HttpResponse {
 
     debug!("Request to delete origin {}", &origin);
 
-    if !check_origin_empty(&req, &origin).unwrap_or(false) {
-        return HttpResponse::new(StatusCode::UNPROCESSABLE_ENTITY);
-    }
-
```

Comment:
Why remove this and the associated function?

Reply:
I started to beef up that function to try to have it check all possible cases but it seemed more likely that we'd miss a case and then have to come back and keep fixing missed cases when in reality if the DB deletion fails 99% of the time it is going to be because of a conflict that we'd want to flag with a 409 anyways.

Reply:
Yeah I guess if the db has foreign key constraints between origins and packages, then trying to delete the origin will fail anyway, so that makes sense.